android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.my_app">
```

Comment:
@sachin-kumar56 , Can we use the proper name for the value of the `package` here?